### PR TITLE
Rename passwordFile to hashedPasswordFile

### PR DESCRIPTION
**Pull request by Pizmovc**
This follows the rename that was done in nixpkgs.

Reference PR https://github.com/NixOS/nixpkgs/pull/254080

### CODE REVIEW

File: pkgs/sops-install-secrets/nixos-test.nix [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
       users.users.example-user = {
         isNormalUser = true;
-        passwordFile = config.sops.secrets.test_key.path;
+        hashedPasswordFile = config.sops.secrets.test_key.path;
```
Comment (Mic92) [inline pkgs/sops-install-secrets/nixos-test.nix:39]: For the 22.03 test we need to make this conditional.

Comment (Pizmovc) [reply pkgs/sops-install-secrets/nixos-test.nix]: Ah yes, of course. I've added a fixup [b43feda](https://github.com/Mic92/sops-nix/pull/433/commits/b43feda77e83656972d2b5e8d4c0d36f74cd8618).

**Comment by Mic92** [comment]
@mergify queue

**Comment by Pizmovc** [comment]
@Mic92 , not sure if you noticed or even if you care about this, but commits did not get squashed before being merged, so now there are two commits on the master branch when there should only be one fixup-ed commit.

**Comment by Mic92** [comment]
Missed it. I can probably use mergify next time to squash it.

**Pull request merged**